crypto.go

```diff
@@ -18,13 +18,24 @@ const PublicKeyBytes = 65
 
 // PublicKey returns the public key for this private key.
 func PublicKey(sk []byte) []byte {
-	x, y := secp256k1.S256().ScalarBaseMult(sk)
```

Comment:
What @whyrusleeping said (sorry for changing directions). Basically, you can use the builtin `elliptic` and `ecdsa` packages (as we were doing before) but with `gitlab.com/yawning/secp256k1-voi`.

Reply:
Example: https://github.com/whyrusleeping/go-did/blob/master/signing.go